package.json

```diff
@@ -47,7 +47,8 @@
     "husky": "^1.2.0",
     "jsbi": "^2.0.5",
     "leap-core": "0.32.4",
-    "solium": "^1.1.8",
+    "regenerator-runtime": "^0.13.2",
```

Comment:
is it needed?

Reply:
as a replacement for `@babel/polyfill`. npm told me so 😁

Reply:
At one point you should just remove the bla bla babel, things. 
It has really no good use for leap-contracts, right?

Reply:
yes, like in solEVM?

Reply:
The only reason we need this here seems because of the `import` instead of using `require` in the `tests/` files. Ya, like in solEVM 😅